File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int n=nums1.length;
        int m=nums2.length;
        int[]merge=new int[m+n];
        System.arraycopy(nums1,0,merge,0,n);
        System.arraycopy(nums2,0,merge,n,m);
        sort(merge);
        int lo=0;
        int hi=merge.length-1;
        double median=0.0;
        if(merge.length%2==0){
            double k=(merge[(lo+hi)/2])+(merge[((lo+hi)/2)+1]);
            median=k/2;
        }
        else{
            median=merge[(lo+hi)/2];
        }
        return median;
        
    }
    public int[] sort(int[]merge){
        for(int i=1;i<merge.length;i++){
            for(int j=0;j<merge.length-i;j++){
                if(merge[j]>merge[j+1]){
                    int temp=merge[j];
                    merge[j]=merge[j+1];
                    merge[j+1]=temp;
                }
            }
            
        }
        return merge;
    }
}

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """
        l=sorted(nums1+nums2)
        n=len(l)
        if(n%2==0):
            return float(l[n//2 -1]+l[n//2])/2
        else:
            return float(l[n//2])
        